chart/values.yaml

```diff
@@ -0,0 +1,160 @@
+# This values.yaml file provides default configuration for Turing Helm chart. 
+# It is also currently used to generate README.md using helm-docs
+# https://github.com/norwoodj/helm-docs
+
+turing:
+  image:
+    # -- Docker registry for Turing API image. User is required to override
+    # the registry for now as there is no publicly available Turing image
+    registry: docker.io/
+    # -- Docker image repository for Turing API
+    repository: turing
+    # -- Docker image tag for Turing API
+    tag: latest
+  # -- Resources requests and limits for Turing API. This should be set 
+  # according to your cluster capacity and service level objectives.
+  # Reference: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+  resources: {}
+  livenessProbe:
+    # -- HTTP path for liveness check
+    path: "/v1/internal/live"
+  readinessProbe:
+    # -- HTTP path for readiness check
+    path: "/v1/internal/ready"
+  service:
+    # -- Turing API Kubernetes service port number
+    externalPort: 8080
+    # -- Turing API container port number
+    internalPort: 8080
+  config:
+    deployment:
+      # -- Maximum wait duration to create or update Turing router
+      timeout: 3m
+      # -- Maximum wait duration to delete Turing router
+      deletionTimeout: 30s
+      # -- Google Cloud Project ID associcated with Turing router
+      gcpProject: ""
+      # -- Environment name associated with Turing router
+      environmentType: id-dev
+      # -- Hard limit on the maximum CPU Turing router can request
+      maxCPU: "8"
+      # -- Hard limit on the maximum memory Turing router can request
+      maxMemory: "8Gi"
+    router:
+      image:
+        # -- Docker registry for Turing router image. User is expected to override
+        # the registry for now as there is no publicly available Turing image
+        registry: docker.io/
+        # -- Docker image repository for Turing router
+        repository: turing-router
+        # -- Docker image tag for Turing router
+        tag: latest
+      # -- Log level for Turing router
+      logLevel: DEBUG
+      # -- Enable debugging for Fiber library used by Turing router
+      fiberDebugLog: true
+      # -- Enable custom metrics
+      customMetrics: true
+      jaeger:
+        # -- Enable Jaeger tracing
+        enabled: false
+        # -- Jaeger tracing collector endpoint 
+        collectorEndpoint: 
+      fluentd:
```

Comment:
I feel like this should be hidden behind a feature toggle. I.e. this configuration should not be required if BQ is not available as an available option for logging. Probably can be addressed in a separate MR

Reply:
Yes, I think we can address this in a pull request regarding configuration format in Turing. Agree that all these plugins and extensions should be a feature toggle.

Reply:
Can we have tracked this in JIRA? either as a new card or a note to existing one about config refactoring

Reply:
Yes sure